File: Data Structures & Algorithms/two-integer-sum/submission-2.cpp
#include <algorithm>

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int i{0};
        for(auto iter{nums.begin()}; iter!=nums.end(); i++,iter++){
            int j{i+1};
            auto iter1 = iter + 1;
            
            for(; iter1!=nums.end(); j++, iter1++)
                if (target-*iter== *iter1)
                return vector<int>{i, j};
        }
    }
};

File: Data Structures & Algorithms/house-robber/submission-2.py
class Solution:
    def rob(self, nums: List[int]) -> int:

        len_lst=len(nums)
        if len(nums)%2: len_lst-=1
        a, b =0,0 
        for i in nums:
            a,b=b, max(b,a+i)

        print(a, b)
        return b